### ISSUE TITLE
Machine-readable output from icommands

### ISSUE THREAD

**Issue by Xophmeister** [irods/irods#4956]
## Feature
It would be very useful if some icommands could have an option for machine-readable (e.g., JSON) output for downstream parsing. Currently, we rely on magical regular expressions that are fragile; an upstream change to the icommands can break everything downstream and it can be hard to find the source of error.

For example, `ienv` and `iuserinfo` would be immediately useful. The output from `ils` and `imeta qu`, I suspect, would also be helpful to many.

**Comment by kript** [comment]
Can we have 'consortium member' tags added please? @Xophmeister is 'one of us'...

**Comment by trel** [comment]
Agreed - we've standardized on JSON for configuration and are getting there with interchange formats as well.  iCommands are definitely on the map.
